mscore/shortcut.cpp

```diff
@@ -4359,7 +4360,25 @@ void Shortcut::resetToDefault()
             }
       source = defaultFileName;
       dirty = true;
-      }
+}
+
+Shortcut* Shortcut::getShortcutByKeySequence(const QKeySequence &keySequence)
+{
+    for (Shortcut* shortcut : _shortcuts.values()) {
+
+        QAction* action = shortcut->action();
+
+        if (!action) {
+            continue;
+        }
+
+        if (action->shortcut() == keySequence) {
+            return shortcut;
+        }
+    }
```

Comment:
Banner style indent of closing braces please. In this case many of those braces are not needed al all, as they bracket a singlc instrunction only